### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by itsmebipul00**
Added navigation and alerts

### CODE REVIEW

**Review by purvasharmadev — APPROVED** [review]
Good to go. code is clean and readable. Adding some comments before the codes will make it more readable , Just a suggestion, else everything looks good.

**Review by srishtigautam21 — APPROVED** [review]
Good to merge

File: Components/Aside/Aside.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  aside ul {
+    height: 90vh;
+    overflow: auto;
+  }
+
+  aside ul::-webkit-scrollbar,
+  article::-webkit-scrollbar {
+    width: 0.5rem;
```
Comment (iRohitGaur) [inline Components/Aside/Aside.css:26]: you can give px values for value less than 10 px

**Review by iRohitGaur — APPROVED** [review]
Great work! Good to merge.

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: Components/Aside/Aside.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  article::-webkit-scrollbar-track {
+    background: hsl(var(--clr-white));
+  }
+
+  aside ul::-webkit-scrollbar-thumb {
+    background: hsl(var(--clr-dark));
+  }
+  article::-webkit-scrollbar-thumb {
```
Comment (sudipkundu999) [inline Components/Aside/Aside.css:40]: 1 line spacing to be added

**Review by sudipkundu999 — APPROVED** [review]
Good to merge

**Comment by itsmebipul00** [comment]
Thanks for the review guys

**Pull request merged**